test/inductor/test_static_cuda_launcher.py

```diff
@@ -272,11 +272,31 @@ def kernel_no_op():
             pass
 
         compiled_kernel = kernel_no_op[(1,)]()
-        self.assertRaisesRegex(
-            NotImplementedError,
-            "No static cuda launcher available",
-            lambda: self._make_launcher(compiled_kernel),
-        )
+        launcher = self._make_launcher(compiled_kernel)
+        device_interface = get_interface_for_device("cuda")
+        stream = device_interface.get_raw_stream(device_interface.current_device())
+        launcher.run(1, 1, 1, stream)
+
+    def test_slow_launch(self):
+        @triton.jit
+        def simple_kernel(arg0, arg1):
+            x = tl.load(arg0)
+            y = arg1
+            tl.store(arg0, x + y)
+
+        arg0 = torch.zeros(1, dtype=torch.int32, device="cuda")
+        arg1 = 5
+        args = (arg0, arg1)
+        compiled_kernel = simple_kernel[(1,)](*args)
+        launcher = self._make_launcher(compiled_kernel)
+        self.assertEqual(arg0, torch.tensor([5], dtype=torch.int32, device="cuda"))
+        self.assertEqual(launcher.arg_tys, "Oi")
+        new_arg0 = torch.zeros(1, dtype=torch.int32, device="cuda")
+        device_interface = get_interface_for_device("cuda")
+        stream = device_interface.get_raw_stream(device_interface.current_device())
+        launcher.slow_launch_kernel = True
```

Comment:
Is this flag only needed for testing?  Maybe we should name it to make it clear?  Could we test the other path with a large number of args to make the code cleaner?

Reply:
Sure, can just add a test with a huge number of args; i'm trying to figure out how to do that lol, because I don't wanna actually manually write a 121 arg function, but I also can't really codegen it because triton needs inspect.get_sourcelints to work. Will figure it out though

Reply:
I think if you use codecache.py's PyCodeCache it should should have source code.